GOVERNANCE.md

```diff
@@ -0,0 +1,46 @@
+# Process for becoming a maintainer
+
+## Your organization is not yet a maintainer
+
+* Express interest to the senior maintainers that your organization is interested in becoming a maintainer. Becoming a maintainer generally means that you are going to be spending substantial time (>25%) on Amundsen for the foreseeable future. Ultimately your goal is to become a senior maintainer that will represent your organization.
+* We will expect you to start contributing increasingly complicated PRs, under the guidance of the existing senior maintainers.
+* We may ask you to do some PRs from our backlog.
+* As you gain experience with the code base and our standards, we will ask you to do code reviews for incoming PRs (i.e., all maintainers are expected to shoulder a proportional share of community reviews).
+* After a period of approximately 2-3 months of working together and making sure we see eye to eye, the existing senior maintainers will confer and decide whether to grant maintainer status or not. We make no guarantees on the length of time this will take, but 2-3 months is the approximate goal.
```

Comment:
```suggestion
* After approximately 2-3 months of working together and making sure we see eye to eye, the existing senior maintainers will confer and decide whether to grant maintainer status or not. We make no guarantees on the length of time this will take, but 2-3 months is the approximate goal.
```

Reply:
Do we need to say `making sure we eye to eye`? I would just rephrase this to read `After about 2-3 months of working on the project, an existing senior maintainer will initiate a voting process to decide whether to award maintainer status. Please note 2-3 months is just a suggested timeline and should by no means considered a guarantee to get maintainer status. Each contributor's timeline may vary widely based on the number and complexity of contributions` -  Please let me know what you think?